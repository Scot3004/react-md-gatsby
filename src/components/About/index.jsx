import React, { Component } from "react";
import { Card, CardText} from "react-md";
import { translate } from "react-i18next";
import UserLinks from "../UserLinks";
import config from "../../../data/SiteConfig";
import "./About.scss";

class About extends Component {
  render() {
    const { t } = this.props;
    return (
      <div className="about-container md-grid mobile-fix">
        <Card className="md-grid md-cell--8">
          <div className="about-wrapper">
            <img
              src={config.userAvatar}
              className="about-img"
              alt={config.userName}
            />
            <CardText>
              <p className="about-text md-body-1">{t("description")}</p>
            </CardText>
            <UserLinks labeled config={config} />
          </div>
        </Card>
      </div>
    );
  }
}

export default translate("about")(About);
