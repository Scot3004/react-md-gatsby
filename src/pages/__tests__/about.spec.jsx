import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import About from '../about';

it('renders correctly', () => {
  const tree = shallow(<About />);
  expect(toJson(tree)).toMatchSnapshot();
});
