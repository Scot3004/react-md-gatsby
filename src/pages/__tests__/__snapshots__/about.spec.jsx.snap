// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="about-container"
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      About | Sergio Orozco
    </title>
    <link
      href="https://www.secorto.com/about/"
      rel="canonical"
    />
  </HelmetWrapper>
  <Translate(About) />
</div>
`;
